pyspecdata/core.py

```diff
@@ -1616,109 +1622,131 @@ def concat(datalist, dimname, chop=False):
             )
         )
     other_info_out = datalist[0].other_info
-    for j in range(0, len(datalist)):
-        # {{{ make list for the shape to check, which contains the dimensions we are NOT concatting along
-        if dimname in shapes[j].dimlabels:
-            newdimsize += shapes[j][dimname]
-            shapetocheck = list(shapes[j].shape)
-            shapetocheck.pop(shapes[j].axn(dimname))
-        else:
+    if dimname in datalist[0].dimlabels:
+        print(
+            "The dimname you specified exists already in one of the datasets so I am assuming it also exists in the other"
+        )
```

Comment:
unsure if needed but I figure useful to tell the user what's going on

Reply:
replaced with assertion